config/locales/views/submissions/en.yml

```diff
@@ -16,11 +16,14 @@ en:
       could_not_collect_released: Could not collect submission because it has already been released.
       could_not_collect_some_due: Could not collect submissions for some groupings and assignment %{assignment_identifier} - the collection date has not been reached for these groupings.
       could_not_collect_some_released: Could not collect submissions for some groupings and assignment %{assignment_identifier} - the submission has already been released.
+      full_overwrite_warning: Collecting and grading this revision will overwrite any previous collections/grading done for this group on this assignment.  Are you sure you want to do this?
+      grading_data_will_be_lost: 'Warning: without selecting this option, all old grading data for the collected submissions will be lost'
       manual_collection: Manual Collection
-      override_existing_html: 'Recollect previously collected submissions? <strong>Warning: any previous grading work will be lost.</strong>'
-      overwrite_warning: Collecting and grading this revision will overwrite any previous collections/grading done for this group on this assignment.  Are you sure you want to do this?
+      override_existing: Recollect previously collected submissions?
```

Comment:
This message should be updated now (since remark data is now also copied)

Reply:
Ah yes, I will actually make this a confirmation message rather than a warning now because we're pretty much copying everything.